### PR TITLE
Social link update

### PR DESCRIPTION
**Pull request by rickycpadilla**
Added a "passage-" prefix to the Passage callback url scheme for Social login. This will prevent issues our server was having with redirect uri's that began with numeric rather than alphabetical character.

### CODE REVIEW

File: Sources/Passage/AuthorizationController/PassageSocialAuthController.swift [hunk 1]
```diff
@@ -70,6 +70,10 @@ final internal class PassageSocialAuthController:
     
     // MARK: STATIC METHODS
     
+    internal static func getCallbackUrlScheme(appId: String) -> String {
```
Comment (jenn-macfarlane) [inline Sources/Passage/AuthorizationController/PassageSocialAuthController.swift:73]: Nice, like the reuse!

File: Sources/Passage/AuthorizationController/PassageSocialAuthController.swift [hunk 2]
```diff
@@ -70,6 +70,10 @@ final internal class PassageSocialAuthController:
...
+        return "passage-\(appId)"
```
Comment (jenn-macfarlane) [inline Sources/Passage/AuthorizationController/PassageSocialAuthController.swift:74]: Just checking - this doesn't need a closing slash or anything? I forget how this works in swift!

**Review by jenn-macfarlane — APPROVED** [review]
Looks good, just one quick question

File: Sources/Passage/AuthorizationController/PassageSocialAuthController.swift (hunk 2 above)
Comment (rickycpadilla) [reply Sources/Passage/AuthorizationController/PassageSocialAuthController.swift:74]: Nope, no closing slash needed :)

**Pull request merged**